index.js

```diff
@@ -15,12 +15,10 @@ Publish.prototype.start = function (options) {
 // Run-type switch
 if (require.main === module) {
   // App called directly
-  module.exports = new Publish().start()
+  module.exports = new Publish().run(true)
```

Comment:
Looks great! The only thing I'd probably change, and that is down to personal taste, is change the `Publish` constructor to accept an options object, rather than individual arguments. This would make the call to it a lot more descriptive.

```js
// Before
module.exports = new Publish().run(true)

// After
module.exports = new Publish().run({isStandalone: true})
```

If the constructor grows to accept more arguments, it helps to keep it short and clean.